scene/animation/animation_mixer.cpp

```diff
@@ -965,6 +957,17 @@ Variant AnimationMixer::_post_process_key_value(const Ref<Animation> &p_anim, in
 	return p_value;
 }
 
+Variant AnimationMixer::post_process_key_value(const Ref<Animation> &p_anim, int p_track, Variant p_value, ObjectID p_object_id, int p_object_sub_idx) {
+	if (is_GDVIRTUAL_CALL_post_process_key_value) {
+		Variant res;
+		if (GDVIRTUAL_CALL(_post_process_key_value, p_anim, p_track, p_value, p_object_id, p_object_sub_idx, res)) {
+			return res;
+		}
+		is_GDVIRTUAL_CALL_post_process_key_value = false;
+	}
+	return _post_process_key_value(p_anim, p_track, p_value, p_object_id, p_object_sub_idx);
+}
+
```

Comment:
Maybe this change would not be necessary if AnimationMixer were registered as GDREGISTER_VIRTUAL_CLASS instead of GDREGISTER_ABSTRACT_CLASS? cc @reduz
This is also applicable to the AnimationNode.

Reply:
It is still necessary. I don't know what is happening in the GDVIRTUAL_CALL method, but I know that it is slow no matter how this class was registered. And so this variable caches the return result of this method if it returned false.

Reply:
Also, for some reason, the performance decreased when the registration was changed to GDREGISTER_VIRTUAL_CLASS